internal/standalone/config/config.go

```diff
@@ -97,6 +97,13 @@ func New(c *cli.Context) (Config, error) {
 }
 
 func (c *Config) Init() (err error) {
+	if c.Template != "" {
```

Comment:
Don't we have to display a warning message in the following case? What do you think?
```
$ trivy --format template  golang:1.12-alpine    
```

In short, it is like this condition.
```
if c.Format == "template" && c.Template == ""{
}
```

Reply:
@knqyf263 Thanks for your review. Added the test case for it in a8bebfd and implemented it in a8be49e.

Reply:
Thanks! I know it is your fault, but I think it is better to add this check for consistency.